### PR TITLE
Fix issue with cpu usage for realtime updates.

### PR DESCRIPTION
**Pull request by serhii-londar**
Enable realtime preview only when feature is nabled.

### CODE REVIEW

**Comment by serhii-londar** [comment]
@winstondu Hi, could you please check whether this fix works for you?

File: CrowdinSDK/Classes/CrowdinSDK/Localization/Extractor/LocalLocalizationExtractor.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ import Foundation
 final class LocalLocalizationExtractor {
     enum Strings: String {
         case LocalPlurals
-        case LocalizableStringsdict
+        case LocalizableStringsdict = "Localizable.stringsdict"
```
Comment (winstondu) [inline CrowdinSDK/Classes/CrowdinSDK/Localization/Extractor/LocalLocalizationExtractor.swift:13]: Why this? Was there a different bug?

**Review by winstondu — APPROVED** [review]
This solve the issue. But also add a comment to `class func swizzle()` for `UILabel+swizzle.swift` and `UIButton+swizzle.swift` that the swizzling is only needed for certain features.

**Comment by serhii-londar** [comment]
@winstondu Comments are added.

File: CrowdinSDK/Classes/CrowdinSDK/Localization/Extractor/LocalLocalizationExtractor.swift (hunk 1 above)
Comment (serhii-londar) [reply CrowdinSDK/Classes/CrowdinSDK/Localization/Extractor/LocalLocalizationExtractor.swift:13]: Not for native SDK, but yes, it fixes a potential issues with extracting local localization)

**Comment by winstondu** [comment]
> @winstondu Comments are added.

Thank you! Please merge and let me know when you have a new release out!

As a side note, do optimize some of the functions like `findKey(for string: String)` when you have a chance in the future.

**Pull request merged**